src/extension/intellisenseHelper.ts

```diff
@@ -126,4 +188,83 @@ export class IntellisenseHelper {
 
         return Q(isRestartRequired);
     }
+
+    /**
+     * Simple check to see if the TypeScript package is in the expected location (where we installed it)
+     */
+    private static isTypeScriptInstalled() : Q.Promise<boolean> {
+        let fileSystem: FileSystem = new FileSystem();
+        let installPath: string = path.join(IntellisenseHelper.getTypeScriptInstallPath(), "lib");
+        return Q.resolve(fileSystem.exists(installPath));
```

Comment:
Since `fileSystem.exists` already returns a `Q.Promise<boolean>` you don't need to wrap it in a `Q.resolve` here